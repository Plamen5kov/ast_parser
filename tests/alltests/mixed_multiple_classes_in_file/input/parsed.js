var MyClass = (function (_super) {
    __extends(MyClass, _super);
    function MyClass() {
        _super.apply(this, arguments);
    }
    MyClass.prototype.MyClassMethod111 = function () {
        console.log("smth");
    };
    MyClass.prototype.MyClassMethod1111 = function () {
        console.log("smth");
    };
    MyClass = __decorate([
        JavaProxy("asd.asd.As111")
    ], MyClass);
    return MyClass;
})(android.view.ViewGroup1111);
var MyButton = android.widget.Button22222222.extend("my.custom.ClassFromOrigina2222222222", {
	onClick222222: function () {
		console.log('click happened');
	},
	onClick222222222: function () {
		console.log('click happened');
	}
});
var MyClass = (function (_super) {
    __extends(MyClass, _super);
    function MyClass() {
        _super.apply(this, arguments);
    }
    MyClass.prototype.MyClassMethod3333 = function () {
        console.log("smth");
    };
    MyClass.prototype.MyClassMethod33333333 = function () {
        console.log("smth");
    };
    MyClass = __decorate([
        JavaProxy("asd.asd.As3333")
    ], MyClass);
    return MyClass;
})(android.view.ViewGroup333333);
(function () {
    var MyButton = android.widget.Button44444444.extend("my.custom.ClassFromOrigina44444", {
        onClick4444: function () {
            console.log('click happened');
        },
        onClick4444444: function () {
            console.log('click happened');
        }
    });
})();
