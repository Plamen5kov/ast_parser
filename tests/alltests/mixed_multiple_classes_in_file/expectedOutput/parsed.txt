Java File: asd.asd.As111 - Extend Class: android.view.ViewGroup1111 - Overridden Methods: MyClassMethod111,MyClassMethod1111
Java File: my.custom.ClassFromOrigina2222222222 - Extend Class: android.widget.Button22222222 - Overridden Methods: onClick222222,onClick222222222
Java File: asd.asd.As3333 - Extend Class: android.view.ViewGroup333333 - Overridden Methods: MyClassMethod3333,MyClassMethod33333333
Java File: my.custom.ClassFromOrigina44444 - Extend Class: android.widget.Button44444444 - Overridden Methods: onClick4444,onClick4444444